src/main/atm2lndType.F90

```diff
@@ -103,7 +104,7 @@ module atm2lndType
      real(r8), pointer :: forc_pbot_downscaled_col      (:)   => null() ! downscaled atm pressure (Pa)
      real(r8), pointer :: forc_rho_downscaled_col       (:)   => null() ! downscaled atm density (kg/m**3)
      real(r8), pointer :: forc_lwrad_downscaled_col     (:)   => null() ! downscaled atm downwrd IR longwave radiation (W/m**2)
-
+     real(r8), pointer :: forc_solad_col                (:,:) => null() ! direct beam radiation (numrad) (vis=forc_sols , nir=forc_soll )
```

Comment:
I'm not sure that this should be changed for UrbanRadiation.

Reply:
> I'm not sure that this should be changed for UrbanRadiation.

I see your point. I don't have strong feelings either way for that one. I thought I had gone through at one point and changed all forcing variables to generally use the downscaled, column-level variables, but now I see that UrbanFluxes still uses the non-downscaled versions, so by analogy UrbanRadiation could also continue to use the non-downscaled versions. I was originally thinking that, as long as the column-level values are initialized to be the same as the gridcell-level values (so that they are the same in places where the downscaling isn't being done), then it would be best to use the column-level values in the physics code in general. But if it feels to you like it's best to continue to use the non-downscaled, gridcell-level values in the urban code, that's fine with me.

Reply:
@swensosc says the forc_solad references have been replaced

Reply:
- [ ] I still see references to the gridcell-level (non-downscaled) forc_solad in a few places in SurfaceRadiation - in some conditionals on use_SSRE. Is this intentional? If so, can you please add a comment explaining why it's correct to use the non-downscaled forc_solad there? If not, can you please change them to use the downscaled versions?

Reply:
- [ ] Thanks a lot for fixing the references to forc_solad in SurfaceRadiation. One more small request: can you please remove the no-longer-used associate statement for `atm2lnd_inst%forc_solad_not_downscaled_grc` in that module so it doesn't turn up in searches?